siac/parse.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+)
+
+// parseSize converts strings of form 10GB to a size in bytes
+// Parsing is flexible in that unnecessary trailing characters are discarded
+// If parsing fails an error is thrown
```

Comment:
Docstring should have full puntuation.

Should also be wrapped to 80 characters. I'm not sure what text editor you are using, but I know vim and sublime text have a command you can use that will auto-wrap/format a block of text to 80 characters. In vim, you highlight the lines you want to wrap, and then press `gq`. I'm not sure what the command is in other text editors.